### ISSUE TITLE
fsck throws: stuck, ran out of copygc reserve!

### ISSUE THREAD

**Issue by marcin-github** [koverstreet/bcachefs#151]
Hi,
after bug described in https://github.com/koverstreet/bcachefs/issues/139#issuecomment-709324506 I did hard reset of vm. After boot I got:
```
[   44.438703] bcachefs (dm-7): IO error: read only
[   44.438739] bcachefs (dm-7): IO error: read only
[   44.438747] bcachefs (dm-7): IO error: read only
[   44.438753] bcachefs (dm-7): IO error: read only
[   44.438766] bcachefs (dm-7): IO error: read only
[   44.438815] bcachefs (dm-7): IO error: read only
[   44.438822] bcachefs (dm-7): IO error: read only
[   44.438828] bcachefs (dm-7): IO error: read only
[   44.438834] bcachefs (dm-7): IO error: read only
[   44.438839] bcachefs (dm-7): IO error: read only
[   52.358619] zram: Added device: zram0
[   52.360343] zram: Added device: zram1
[   52.361446] zram: Added device: zram2
[   52.363017] zram: Added device: zram3
[   52.445322] zram2: detected capacity change from 0 to 8970567680
[   52.620030] EXT4-fs (zram2): mounted filesystem without journal. Opts: discard,discard
[   52.651986] zram3: detected capacity change from 0 to 12582912
[   52.743193] Adding 12284k swap on /dev/zram3.  Priority:16383 extents:1 across:12284k SSDscFS
[   69.708881] bch2_write: 10192 callbacks suppressed
```

fsck can't fix it:
```
# bcachefs fsck -y /dev/vg-ogolny/rrd
journal read done, 59490 keys in 16 entries, seq 16658613
starting mark and sweep
starting fsck
stuck, ran out of copygc reserve!
mounted with opts: metadata_replicas=3,data_replicas=2,compression=zstd,wide_macs,usrquota,grpquota,prjquota,noreflink,degraded,fsck,fix_errors
emergency read only
```

**Comment by koverstreet** [comment]
Could you take a dump for me?

**Comment by koverstreet** [comment]
Fixed by 0568ed4886
